exporter/clickhouseexporter/exporter_logs.go

```diff
@@ -192,7 +192,7 @@ func createDatabase(cfg *Config) error {
 		return nil
 	}
 	// use default database to create new database
-	dsnUseDefaultDatabase := strings.Replace(cfg.DSN, database, defaultDatabase, 1)
+	dsnUseDefaultDatabase, _ := getDefaultDns(cfg.DSN, database)
```

Comment:
can you do something with the error returned?

Reply:
done